### ISSUE TITLE
HTTP Status Code 411 - Length Required

### ISSUE THREAD

**Issue by tkurmann** [Corwynt/atom-thesaurus#2]
HTTP Status Code 411 - Length Required error appears when using bighugelabs.

Change line 58 in main.js to 

`method: "POST",`

and it works again.

**Comment by Corwynt** [comment]
For me it is still working and the API documentation states nothing about POST Requests(http://words.bighugelabs.com/api.php). 

Does it happen with any selected word?

**Comment by tkurmann** [comment]
Yes, it happens with every word. I agree that POST may not be the best option, but it is working fine for me on Atom v1.13.1 / ubuntu 16.10. What also works is adding 

`            headers: {
              'Content-Length': '0'
            },`
while using GET. It seems my version of request-promise loses the content-length header.
